includes/discovery/os/zxr10.inc.php

```diff
@@ -4,4 +4,7 @@
     if (preg_match('/^ZXR10/', $sysDescr)) {
         $os = 'zxr10';
     }
+    if (str_contains($sysDescr, "ZTE Ethernet Switch")) {
```

Comment:
The preg_match was specifically doing it on sysDescr starts explicitly with ZXR10 as it's a small non-specific match